temporaltest/server.go

```diff
@@ -103,6 +103,14 @@ func (ts *TestServer) GetDefaultNamespace() string {
 	return ts.defaultTestNamespace
 }
 
+// GetFrontendHostPort returns the host:port for this server.
+//
+// When constructing a Temporal client from within the same process,
+// GetDefaultClient or NewClientWithOptions should be used instead.
+func (ts *TestServer) GetFrontendHostPort() string {
```

Comment:
The Go convention is to not use `Get` prefixes for getters. I see we've already violated that in this file so I'd accept your change as-is.

```suggestion
// FrontendHostPort returns the host:port for this server.
//
// When constructing a Temporal client from within the same process,
// GetDefaultClient or NewClientWithOptions should be used instead.
func (ts *TestServer) FrontendHostPort() string {
```

Reply:
Yes, I stuck with the Get prefix for consistency within the package. If you prefer no Get prefix so that the interface is more similar to the DevServer, we could do that instead though.